File: CinemaRest.Service/CinemaRest.Service/Models/Link.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CinemaRest.Service.Models
{
    /// <summary>
    /// Klasa reprezentująca odnośnik do zasobu
    /// </summary>
    public class Link
    {
        /// <summary>
        /// URI do zasobu
        /// </summary>
        public string Href { get; set; }
        /// <summary>
        /// Nazwa zasobu
        /// </summary>
        public string Rel { get; set; }
        /// <summary>
        /// Metoda HTTP
        /// </summary>
        public string Method { get; set; }

        public Link() { }

        public Link(string href, string rel, string method)
        {
            Href = href;
            Rel = rel;
            Method = method;
        }
    }
}

File: CinemaRest.Client/src/cinemarest/client/models/Link.java
package cinemarest.client.models;

public class Link {
    private String rel;
    private String href;
    private String method;

    public Link() {}

    public Link(String href) {
        this.href=href;
    }

    public String getRel() {
        return rel;
    }

    public void setRel(String rel) {
        this.rel = rel;
    }

    public String getHref() {
        return href;
    }

    public void setHref(String href) {
        this.href = href;
    }

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }
}
